mitosheet/src/mito/utils/keyboardShortcuts.tsx

```diff
@@ -128,6 +128,67 @@ export const keyboardShortcuts: KeyboardShorcut[] = [
         action: ActionEnum.OpenFindAndReplace
     },
     {
+        macKeyCombo: {
+            altKey: true,
+            keys: ['F1']
+        },
+        winKeyCombo: {
+            altKey: true,
+            keys: ['F1']
+        },
+        action: ActionEnum.Graph
+    },
+    {
+        macKeyCombo: {
+            metaKey: true,
+            keys: ['o']
+        },
+        winKeyCombo: {
+            ctrlKey: true,
+            keys: ['o']
+        },
+        preventDefaultAndStopPropagation: true,
+        action: ActionEnum.Import_Files
+    },
```

Comment:
On safari, I'm not sure this is working. Command + O opens the open file menu.

Reply:
Is the focus on the mitosheet?